core/app/[locale]/(default)/error.tsx

```diff
@@ -0,0 +1,5 @@
+'use client';
+
+import { lazy } from 'react';
+
+export default lazy(() => import('../_components/error'));
```

Comment:
How come we're doing this? Next.js files are already lazily loaded so this shouldn't be necessary

Reply:
We should define a new component here and render the error component so it's clear we're passing `retry` as a function and not a server action